app/views/shipit/stacks/show.html.erb

```diff
@@ -23,7 +23,7 @@
     <div class="banner banner--red">
       <div class="banner__inner wrapper">
         <div class="banner__content">
-          <h2 class="banner__title">This repository seems to be missing!</h2>
+          <h2 class="banner__title">This repository or branch seems to be missing!</h2>
```

Comment:
We could even go as far as mentioning the branch: `@stack.branch`.

But that will do.